### DISCUSSION TITLE
Deferred update of property on related model fails

### DISCUSSION THREAD

**Discussion by goldmerc** [livewire/livewire#5020] [7. Bugs (archived)]
livewire/livewire: v2.10.5
laravel/framework: v9.17.0

I have an input wired to update a field in a related model. 

Without deferring the property updates just fine and then I can save the updated model and relations and see the updates working in the database.

If I turn on defer, so that the syncInput + callMethod updates are in the same request payload, I get an SQL error...

`SQLSTATE[42S22]: Column not found: 1054 Unknown column 'RELATIONSHIP_NAME' in 'field list'
`

So, it seems like when the wired property of a related model is deferred, Livewire is not updating the related table but rather treating the relation as if it was a field in the table of the parent model. 

The update works fine if deferral is off and syncInput + callMethod are in different requests.

I can't figure out if this is a bug or something I'm doing wrong?

**Comment by joshhanley** [comment]
@goldmerc please share your code so we can have a look 🙂

When you paste code into github, you need to surround it in triple back ticks and then specify the file type like php or blade.

For example
\```php
\\ PHP code here
\```

\```blade
\\ HTML and blade code here
\```
